src/main/resources/application.properties

```diff
@@ -7,4 +7,11 @@ spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.jpa.show-sql=true
 spring.jpa.hibernate.naming.physical-strategy=org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
 
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQLDialect
+token.signing.key=413F4428472B4B6250655368566D5970337336763979244226452948404D6351
```

Comment:
not sure what is this? for signing JWTs?

Reply:
line 10: i already remove ( i was trying to solve issue with DBConnection, but it was not solution), 
line 11: it is security key for signing Jwt.